### ISSUE TITLE
new template for responsive YT video

### ISSUE THREAD

**Issue by zonky2** [contao-bootstrap/templates#21]
new template
```diff
<?php $this->extend('block_unsearchable'); ?>

<?php $this->block('content'); ?>
-    <iframe <?= $this->size ?> src="<?= $this->src ?>" allowfullscreen></iframe>
+  <div class="embed-responsive embed-responsive-16by9">
+    <iframe class="embed-responsive-item"<?= $this->size ?> src="<?= $this->src ?>" allowfullscreen></iframe>
+  </div>

<?php $this->endblock(); ?>
```
The class `embed-responsive-16by9` should be passed from the outer div with ce_youtube to the iframe-enclosing div

**Comment by dmolineus** [comment]
There should be probably different templates for each ratio

**Comment by zonky2** [comment]
> There should be probably different templates for each ratio

I find rather inflexible...

**Comment by dmolineus** [comment]
What do you suggest?

**Comment by zonky2** [comment]
we can use the same "magic function" like "img-fluid" to transfer the css class to the right div container...

**Comment by dmolineus** [comment]
Since https://github.com/contao/core-bundle/commit/0d0aba2274ff3d350bda6e2170f710a5d931ed51#diff-800975046c3ea232e3dd2a377584899b Contao provides an aspect setting. I prefer using it with a customized template. Only drawback is, that's not available in Contao 4.4.

**Comment by zonky2** [comment]
o.k. - maybe we can use "embed-responsive-16by9" as default and everybody can override this template (and css class)

**Comment by dmolineus** [comment]
I decided to backport the `playerAspect` setting so it's available in Contao 4.4 as well, see a861b1ce621ddfa9432bcfebc5dc7391d4345c8d
